cli/commands/create/create.go

```diff
@@ -123,9 +123,26 @@ func ParseResources(in io.Reader) ([]types.Resource, error) {
 			resources = append(resources, w.Value)
 		}
 	}
+
+	// TODO(echlebek): remove this
+	filterCheckSubdue(resources)
+
 	return resources, err
 }
 
+// filterCheckSubdue nils out any check subdue fields that are supplied.
+// TODO(echlebek): TODO(echlebek): TODO(echlebek): this is temporary; remove it after fixing check subdue.
```

Comment:
Wow, I hope you don't forget to remove it.

Reply:
o_O

Reply:
Amended the stutter